options/options.cc

```diff
@@ -321,7 +321,12 @@ void ColumnFamilyOptions::Dump(Logger* log) const {
                      compaction_options_fifo.allow_compaction);
     std::string collector_names;
     for (const auto& collector_factory : table_properties_collector_factories) {
+      std::string collector_config;
```

Comment:
We could generate the entire string that gets logged using a single `std::ostringstream`. I think that would be cleaner and a bit more efficient. (Also, `collector_names` is a bit of a misnomer now since we log the entire collector configuration.)